azext_iot/sdk/digitaltwins/dataplane/azure_digital_twins_api.py

```diff
@@ -46,11 +52,12 @@ def __init__(
 
         super(AzureDigitalTwinsAPIConfiguration, self).__init__(base_url)
 
-        self.add_user_agent('azuredigitaltwinsapi/{}'.format(VERSION))
+        self.add_user_agent('azure-mgmt-device/{}'.format(VERSION))
```

Comment:
I think the old user-agent was more descriptive, any reason we changed this?

Reply:
dammit - every time I use the autofill stuff